src/components/Header.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
-import styled from 'styled-components'
 import { Link } from 'react-router-dom'
+import styled from 'styled-components'
 import imagePathBackArrow from '../images/back-arrow.svg'
 
-export default function Header({ children, hasBackArrow }) {
+export default function Header({ children, extraElements, hasBackArrow }) {
```

Comment:
Was sind `extraElements`?

Reply:
Hab es jetzt in extraComponents umbenannt. Das sind weitere Elemente, die dem Header hinzugefügt werden können. In dem Fall ist es bei der RecipeDetailPage der AddOrRemoveFromMealplan-Toggler.